rock_paper_scissors/api/models.py

```diff
@@ -0,0 +1,64 @@
+from sqlalchemy import Column, Integer, String
+from sqlalchemy import Enum
+from sqlalchemy import ForeignKey
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+
+
+ModelBase = declarative_base()
+
+
+class Game(ModelBase):
+    __tablename__ = 'game'
+
+    id = Column(Integer, primary_key=True)
+    player_1_id = Column(Integer, ForeignKey('user.id'))
+    player_2_id = Column(Integer, ForeignKey('user.id'))
+    round_id = Column(Integer, ForeignKey('round.id'))
+    rounds_choice = Column(Integer)
+    current_round = Column(Integer)
+    winner_id = Column(Integer, ForeignKey('user.id'))
+
+    player_1 = relationship('User', foreign_keys=[player_1_id])
+    player_2 = relationship('User', foreign_keys=[player_2_id])
+    winner = relationship('User', foreign_keys=[winner_id])
+    round = relationship('Round')
+
+
+class Round(ModelBase):
+    __tablename__ = 'round'
+
+    id = Column(Integer, primary_key=True)
+    player_1_choice_id = Column(Integer, ForeignKey('choice.id'))
+    player_2_choice_id = Column(Integer, ForeignKey('choice.id'))
+    winner_id = Column(Integer, ForeignKey('user.id'))
```

Comment:
I wonder if this should instead be `winning_choice_id`, and a FK to `choice.id`.

The reason for this is that we can always look to the `Choice.player_id` value to see the `User` who made the winning choice. Otherwise we'd need some logic to ensure that `winner_id` actually belongs to one of the two players for the round.